tests/integration/issues/github_241/test_topk.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import numpy as np
+import pytest
+from jina.flow import Flow
```

Comment:
it would be nice if we can respect import sequence as in core, such as
```python
import os

import pytest
import numpy as np

from jina...
```